src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,44 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("Mate Java");
+        book.setPrice(BigDecimal.valueOf(39.99));
+
+        Book book2 = bookDao.create(book);
```

Comment:
Ensure that the `create` method in `BookDao` is implemented to handle the creation of a `Book` object and returns the created `Book` with a valid ID.